src/sagas/console/vvFileUtils.js

```diff
@@ -21,14 +21,17 @@ export function adjustVVFile ({ data, options, usbFilter, isSpice }) {
   }
 
   if (usbFilter) {
-    data = data.replace(/^\[virt-viewer\]$/mg, `[virt-viewer]\nusb-filter=${usbFilter}`)
+    data = data.replace(/^usb-filter=.*$/mg, `usb-filter=${usbFilter}`)
   }
 
   if (options && isSpice) {
     const smartcardEnabled = options.get ? options.get('smartcardEnabled') : options.smartcardEnabled
     data = data.replace(/^enable-smartcard=[01]$/mg, `enable-smartcard=${smartcardEnabled ? 1 : 0}`)
   }
 
+  // make USB Auto-Share to be always enabled in VM Portal, VM portal is not aware of VM's console options settings
+  data = data.replace(/^enable-usb-autoshare=.*$/mg, 'enable-usb-autoshare=1')
```

Comment:
Hmm maybe.. I used the constant because of the https://bugzilla.redhat.com/show_bug.cgi?id=1811466#c6 It seemed to me quite straightforward about how we want to have it fixed.

Reply:
True, and it should meet the base requirements of the BZ.

But...I looked at webadmin on the Console Options dialog.  The "autoshare USB" was disabled and checked for SPICE.  Leads me to believe that the checkbox is driven by that config value.  It shouldn't be too much work to be able to pull that config key and set the auto-share value to it.  I'd be surprised if anyone would set it to false, but for sure if an admin tried to set it to false, and it didn't take effect on VM Portal, there would be a ticket opened.

Reply:
After some investigation, this is becoming more complicated. To make USB autoshare work for a specific VM, I found that we have to make two things, in webadmin:
- enable "USB Support" in Console options of a specific VM
- in Console Options: select SPICE console and check "Enable USB Auto-Share" in "Spice Options".
What exactly `enable-usb-autoshare` in the console.vv file should represent? Which option(s)?